examples/pytorch/audio-classification/run_audio_classification.py

```diff
@@ -41,6 +41,9 @@
 from transformers.utils import check_min_version, send_example_telemetry
 from transformers.utils.versions import require_version
 
+# Initialize MLFlow
+import mlflow
+mlflow.set_tracking_uri("http://127.0.0.1:5000")
```

Comment:
Shouldn't `tracking_uri` be a parameter with default value?
Or can we convince that this script will always use the same uri regardless of future change?

For example, in the future we can use external cloud for mlflow, or our remote server (other than script-running-server). If URI is already configurable or still fine for future changes, then please ignore this comment

Reply:
Model Runner sets up 'MLFLOW_TRACKING_URI'.
There's no need to call set_tracking_uri in a training/inference script.